### PR TITLE
DP-2238: Add JS to indigo3 library

### PR DESCRIPTION
**Pull request by benjiebeetle**

### CODE REVIEW

**Comment by MarBuc** [comment]
@azy141 this is the second PR that you need to start to work on your `Drupal behaviours` story?
@gideoncresswell is this correct Gideon?

if so, could we give it a go at reviewing/approving it?

**Comment by azy141** [comment]
I've removed everyone whilst I test if this will work locally

**Review by azy141 — APPROVED** [review]

File: js/appsrc/carousel.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+$(document).ready(function(){
+// DO WE NEED THIS GLOBAL VARIABLE ANY MORE, OR CAN WE BE SMARTER?
```
Comment (gideoncresswell) [inline js/appsrc/carousel.js:2]: Can these comments go?

File: js/appsrc/carousel.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                },
+                1000:{
+                    items:5,
+                    nav:true,
+                    loop:false
+                }
+            }
+        });
+
+/*        if (typeof $.fn.owlCarousel == "undefined") {
```
Comment (gideoncresswell) [inline js/appsrc/carousel.js:73]: And these?

File: js/app/accordion.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                $(this).next(".accordion__description").addClass("active").slideDown();
+        	    closeAllInactiveAccordionPanels();
+            }
+        });
+	}
+
+    initAccordions();
+}
+};
+})(jQuery, Drupal);
```
Comment (gideoncresswell) [inline js/app/accordion.js:40]: New line at the end.

File: js/app/lightbox.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            $('.lightbox').remove();
+        });
+    }
+
+    $(window).resize(function() {
+        size = window.getComputedStyle(document.body,':after').getPropertyValue('content');
+    });
+}
+};
+})(jQuery, Drupal);
```
Comment (gideoncresswell) [inline js/app/lightbox.js:59]: New line at the end

File: js/app/tabs.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      }
+
+    };
+
+    Tabs.init();
+    // This adds some style to the tabs. Check with Louise/Joel about use.
+    $('body').addClass('js');
+}
+};
+})(jQuery, Drupal);
```
Comment (gideoncresswell) [inline js/app/tabs.js:62]: New line at the end

**Review by azy141 — APPROVED** [review]

**Pull request merged**